enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -27,7 +27,8 @@ const _StaticDashboard = ({
 }: {
   dashboardId: DashboardId;
   parameterQueryParams: Query;
-} & Partial<EmbedDisplayParams>) => {
+  hideParameters: string[];
```

Comment:
I added this since our dashboards currently take in a comma-separated string, but I thought this was silly to pass as a prop. It'll make more sense for a user to use a string array that we handle on our side